package.json

```diff
@@ -2,7 +2,20 @@
   "name": "opg-sirius-header",
   "version": "1.0.0",
   "license": "MIT",
-  "dependencies": {
+  "scripts": {
+    "serve": "http-server",
+    "compile-sass": "node-sass sass/main.scss css/all.css",
+    "cypress": "npx cypress open",
+    "cypress-headless": "cypress run --headless"
+  },
+  "devDependencies": {
+    "@ministryofjustice/frontend": "^1.0.0",
+    "cypress": "^12.15.0",
+    "govuk-frontend": "^4.3.1",
+    "node-sass": "^9.0.0",
```

Comment:
You probably want just `sass` (which is dart sass under the hood) rather than the node one. Node sass is due to be deprecated so won't get updates/patches.

Reply:
Thanks Matt